site-src/v1alpha2/references/policy-attachment.md

```diff
@@ -0,0 +1,294 @@
+# Policy Attachment
+
+Policies attached to Gateway API resources and implementations should use the
+following approach to ensure consistency across implementations of the API.
+There are three primary components of this pattern:
+
+* A shared means of attaching policy to resources.
+* Support for configuring both default and override values within policy
+  resources.
+* A shared hierarchy to illustrate how default and override values should
+  interact.
+
+This kind of standardization will not only enable consistent patterns, it will
+enable tooling such as kubectl plugins to be able to visualize all policies that
+have been applied to a given resource.
+
+## Policy Attachment for Ingress
+Attaching policy to Gateway resources for ingress use cases is relatively
+straightforward. A policy can reference the resource it wants to apply to.
+Access is granted with RBAC - anyone that has access to create a RetryPolicy in
+a given namespace can attach it to any resource within that namespace.
+
+![Simple Ingress Example](/geps/images/713-ingress-simple.png)
```

Comment:
404

Reply:
Not sure how to make this work on github previews + docs site, but deploy preview makes it look like it works there at least: https://deploy-preview-736--kubernetes-sigs-gateway-api.netlify.app/v1alpha2/references/policy-attachment/#policy-attachment-for-ingress.